dfusion

```diff
@@ -15,6 +13,18 @@ def main():
     """
     pass
 
+
+@main.command()
+@click.option('--count', default=100, help='Number of tokens to return, used for pagination')
+@click.option('--skip', default=0, help='Number of tokens to skip, used for pagination')
+@click.option('--sort', default="symbol", help='Sort result by a field, used for pagination')
+@click.option('--format', default="pretty", help='Format type i.e. pretty, csv')
+@click.option('-v', '--verbose', count=True)
+def tokens(count, skip, sort, format, verbose):
+    """Get tokens listed in dFusion. For more details, see https://docs.gnosis.io/dfusion/docs/addtoken1"""
+    get_tokens(count=count, skip=skip, sort=sort, format=format, verbose=verbose)
```

Comment:
My python is rusty but isn't it the same as
```python
get_tokens(count, skip, sort, format, verbose)
```
because of ordering. Not that you need to change, being explicit is good

Reply:
yes, it's enough. I just made for not being too order dependent, cause there was already many arguments

Happy to change it if you think is better to take other approach.
As the other comment. I'll merge so I can start new PRs from this one, but happy to review this in future PRs.